features/makepot.feature

```diff
@@ -2911,6 +2911,52 @@ Feature: Generate a POT file of a WordPress project
       msgid "Bar"
       """
 
+  Scenario: Extract strings from a Blade-PHP file
+    Given an empty example-project directory
+    And a example-project/stuff.blade.php file:
+      """
+	  @php
+		__('Test');
+	  @endphp
+      @extends('layouts.app')
+	  
+	  @php(__('Another test.', 'domain'))
+
+      @section('content')
+        @include('partials.page-header')
+
+        @if (! have_posts())
+          <x-alert type="warning">
+            {!! __('Page not found.', 'foo-plugin') !!}
+          </x-alert>
+
+          {!! get_search_form(false) !!}
+        @endif
+      @endsection
+      """
+
+    When I try `wp i18n make-pot example-project result.pot --ignore-domain --debug`
```

Comment:
An additional scenario for testing extraction without `--ignore-domain` would be helpful too.